mods/ra/rules/defaults.yaml

```diff
@@ -171,10 +171,11 @@
 	DamageMultiplier@IRONCURTAIN:
 		RequiresCondition: invulnerability
 		Modifier: 0
-	TimedConditionBar:
-		Condition: invulnerability
-	ExternalCondition@INVULNERABILITY:
+	GrantConditionOnPrerequisite:
 		Condition: invulnerability
+		Prerequisites: global-invulnerability
+	DrainPrerequisitePowerOnDamage:
```

Comment:
What effect should this have in the testcase? I didn't notice any difference to the IC draining speed when shooting at my own units and buildings.

Reply:
Test with something that damages your units, can recommend to nuke your base.

Reply:
You notice the difference when you test it with something that deals a lot of damage, like a nuke.

Reply:
Ah, I see.